services/network_config_manager/src/main/java/com/futurewei/alcor/netwconfigmanager/entity/ResourceMeta.java

```diff
@@ -0,0 +1,29 @@
+package com.futurewei.alcor.netwconfigmanager.entity;
+
+import com.futurewei.alcor.schema.Goalstate;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ResourceMeta {
+
+    private String ownerId;
+    private List<Goalstate.ResourceIdType> resources;
+
```

Comment:
@xieus What's the ownerId here? projectID?

Reply:
It depends. It can be private ip or host id, depending on the scenario.